Assignments/TestFour.java

```diff
@@ -0,0 +1,105 @@
+package assignments.qatechhub;
+
+import org.testng.annotations.Test;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Parameters;
+
+import java.util.Iterator;
+import java.util.List;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.firefox.FirefoxDriver;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.Select;
+import org.testng.annotations.AfterClass;
+
+public class TestFour {
+  WebDriver driver;
+  
+  @Parameters({ "Browser" })
+  @BeforeClass
+  public void openEbay(String browser) {
+	  if(browser.equalsIgnoreCase("Chrome"))
+	  {
+		  WebDriverManager.chromedriver().setup();
+		  driver = new ChromeDriver();
+	  }
+	  else if(browser.equalsIgnoreCase("Firefox")) {
+		  WebDriverManager.firefoxdriver().setup();
+		  driver = new FirefoxDriver();
+	  }
+	  else {
+		  System.out.println("Wrong Choice");
```

Comment:
instead of writing this in all test classes 
write one base class , extend it and use function from there or using object creation